scripts/zones/Phomiuna_Aqueducts/npcs/_0rn.lua

```diff
@@ -12,20 +12,16 @@ end
 
 entity.onTrigger = function(player, npc)
 
-    local DoorOffset = npc:getID()
+    local doorOffset = npc:getID()
 
-    player:messageSpecial(ID.text.LAMP_OFFSET+2) -- water lamp
-    npc:openDoor(7) -- lamp animation
+    player:messageSpecial(ID.text.LAMP_OFFSET+7) -- Dark lamp
+    npc:openDoor(7) -- Lamp animation
 
     local day = VanadielDayOfTheWeek()
 
-    if (day == xi.day.WATERSDAY) then
-        if (GetNPCByID(DoorOffset+7):getAnimation() == 8) then -- lamp fire open?
-            GetNPCByID(DoorOffset-2):openDoor(15) -- Open Door _0rk
-        end
-    elseif (day == xi.day.LIGHTNINGDAY) then
-        if (GetNPCByID(DoorOffset+2):getAnimation() == 8) then -- lamp lightning open?
-            GetNPCByID(DoorOffset-2):openDoor(15) -- Open Door _0rk
+    if day == xi.day.LIGHTSDAY or day == xi.day.DARKSDAY then
```

Comment:
Multi conditional statements should look like:

if 
    day == xi.day.LIGHTSDAY or 
    day == xi.day.DARKSDAY
then